rc/lsp.kak

```diff
@@ -1415,6 +1417,26 @@
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-breadcrumbs-request -docstring "request updating lsp_modeline_breadcrumbs for the buffer" %{
+  nop %sh{
+    window_start_line="$(echo $kak_window_range | cut -d' ' -f1)"
```

Comment:
unused, removing. It's not clear whether cursor or viewport is better, and cursor is definnitely simpler